setup.py

```diff
@@ -14,6 +14,7 @@
 
 install_requires = [
     'draftjs_exporter==0.7.0',
+    'jinja2>=2.9.5,<3'  # TODO: remove when it's not necessary (https://github.com/springload/wagtaildraftail/issues/13)
```

Comment:
As far as I know, at the moment this package supports both Jinja2 and Django templates. Is it a good idea to force the install of Jinja2 in here?

I also realise that jinja2 is required in `filters.py`, and somehow I think I still got this to work in the test site in `tests` without having it installed. What's going on?